provisioning/service/src/Config/AttestationMechanism.cs

```diff
@@ -129,111 +82,47 @@ private AttestationMechanism(AttestationMechanismType type, TpmAttestation tpm,
         /// <returns>The attestation of this object.</returns>
         public Attestation GetAttestation()
         {
-            switch (Type)
+            return Type switch
```

Comment:
This is pretty neat

Reply:
yeah, a newer C# feature. I'm not all the way used to it, but getting more used to seeing it.